web-common/src/main/java/org/openmrs/eip/web/contoller/BaseRestController.java

```diff
@@ -52,6 +53,31 @@ public Object doGet(Long id) {
 		    "jpa:" + getName() + "?query=SELECT c FROM " + getName() + " c WHERE c.id = " + id, null, getClazz());
 	}
 	
+	public Map<String, Object> getByDateCreated(SenderSearch search) {
+
+	        Map<String, Object> results = new HashMap<String, Object>(2);
+	        List<Object> items;
+
+	        if (search.getStartDate().isBlank() || search.getEndDate().isBlank()) {
+	            items = on(camelContext).to("jpa:" + getName() + "?query=SELECT c FROM " + getName()
+	                    + " c &maximumResults=" + DEFAULT_MAX_COUNT).request(List.class);
+	        } else {
+	            items = on(camelContext).to("jpa:" + getName() + "?query=SELECT c FROM " + getName()
+	                    + " c  where c.dateCreated > '" + search.getStartDate() + "' and c.dateCreated < '"
```

Comment:
I think it should be `>=` and `<=` which will allow one to search items on a single date

Reply:
Done